accounts-service/com.endlessm.ParentalControls.policy

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<policyconfig>
+  <action id="com.endlessm.ParentalControls.AppFilter.ChangeOwn">
+    <description>Change own app filter</description>
```

Comment:
Is it only the `<message>` that gets shown to a human? If `<description>` is shown, I think:

> Read your own app filter

would be clearer.

Reply:
I don’t know which get shown to a human, but certainly both the `<description>` and `<message>` are translated in the `.policy` files on my machine. I’ll tweak the string, thanks for the suggestion.